src/ZendeskApi_v2/Models/Shared/Result.cs

```diff
@@ -22,5 +21,14 @@ public class Result
 
         [JsonProperty("status")]
         public string Status { get; set; }
+
+        [JsonProperty("index")]
+        public int Index { get; set; }
```

Comment:
please make this a long.  see https://developer.zendesk.com/api-reference/ticketing/introduction/#data-types

```suggestion
        public long Index { get; set; }
```

Reply:
Ah derp. Not sure how I missed that. Apologies.